packages/sync/src/Listener.php

```diff
@@ -188,6 +188,11 @@ public function can_add_to_queue( $queue ) {
 	 * @param mixed ...$args Args passed to the action.
 	 */
 	public function full_sync_action_handler( ...$args ) {
+		if ( Settings::get_setting( 'full_sync_send_immediately' ) ) {
```

Comment:
Isn't this going to trigger lots of additional requests? Maybe that's fine but... I just wanted to check.

Reply:
this is hooked only to `full_sync_start` `full_sync_end` and `full_sync_cancel`